src/dmd/cli.d

```diff
@@ -328,18 +328,19 @@ struct Usage
     {
         string bugzillaNumber; /// bugzilla issue number (if existent)
         string name; /// name of the transition
-        string paramName; // internal transition parameter name
-        string helpText; // detailed description of the transition
+        string paramName; /// internal transition parameter name
+        string helpText; /// detailed description of the transition
+        bool deprecated_; /// whether the flag is still in use
     }
 
     /// Returns all available transitions
     static immutable transitions = [
         Transition("3449", "field", "vfield",
             "list all non-mutable fields which occupy an object instance"),
         Transition("10378", "import", "bug10378",
-            "revert to single phase name lookup"),
+            "revert to single phase name lookup", true),
         Transition(null, "checkimports", "check10378",
-            "give deprecation messages about 10378 anomalies"),
+            "give deprecation messages about 10378 anomalies", true),
```

Comment:
As there's no `Flag` available here, I was also thinking about doing the following for better clarity:

```
enum Deprecated { no, yes}
....
"revert to single phase name lookup", Transition.Deprecated.yes),
```

Reply:
I started to work on something similar, and went with setting the `paramName` as `null` to mean `deprecated`